package-scripts/datadog-agent/postinst

```diff
@@ -39,8 +39,13 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
           configure)
               update-rc.d datadog-agent defaults
               update-rc.d datadog-agent enable >/dev/null 2>&1
-              adduser --system dd-agent --disabled-login --shell /bin/sh --no-create-home --quiet
-              usermod -d /opt/datadog-agent dd-agent
+              # Only add dd-agent user if it doesn't already exist
+              id -u dd-agent &>/dev/null
```

Comment:
If you really want that line to ouput nothing you should change that to `id -u dd-agent >/dev/null 2>&1` :)